packages/components/src/alignment-matrix-control/utils.tsx

```diff
@@ -53,11 +69,26 @@ export function transformValue( value: AlignmentMatrixControlValue ) {
  */
 export function getItemId(
 	prefixId: string,
-	value: AlignmentMatrixControlValue
+	value: VoidableAlignmentMatrixControlValue
 ) {
-	const valueId = transformValue( value ).replace( ' ', '-' );
+	const valueId = transformValue( value )?.replace( ' ', '-' );
+	return valueId && `${ prefixId }-${ valueId }`;
+}
 
-	return `${ prefixId }-${ valueId }`;
+/**
+ * Extracts an item value from its ID
+ *
+ * @param prefixId An ID prefix to remove
+ * @param id       An item ID
+ * @return         The item value
+ */
+export function getItemValue(
+	prefixId: string,
+	id: VoidableAlignmentMatrixControlValue
+) {
+	return transformValue(
```

Comment:
We're using `transformValue` for an ID and not for a value? Might be a sign we have to reconsider naming.

Reply:
I think in this instance it's actually just a bit confusing. We're accepting an ID, smooshing it into a value-shaped blob, then transforming that.

This might make it clearer...

```js
export function getItemValue(
  prefixId: string,
  id: VoidableAlignmentMatrixControlValue
) {
  const value = id?.replace( prefixId + '-', '' ) as VoidableAlignmentMatrixControlValue;
  return normalizeValue(value);
}
```